website/payments/admin_views.py

```diff
@@ -40,10 +40,20 @@ def post(self, request, *args, app_label, model_name, payable, **kwargs):
         payable_model = apps.get_model(app_label=app_label, model_name=model_name)
         payable_obj = payable_model.objects.get(pk=payable)
 
-        result = services.create_payment(
-            payable_obj, self.request.member, request.POST["type"],
-        )
-        payable_obj.save()
+        try:
+            result = services.create_payment(
+                payable_obj, self.request.member, request.POST["type"],
+            )
+            payable_obj.payment = result
+            payable_obj.save()
+        # pylint: disable=broad-except
+        except Exception as e:
+            messages.error(
```

Comment:
Does this propagate the error to the sentry exception logging?

Reply:
Oeh, good catch, it doesn't... What do you think, is it better to just crash the site and send it to sentry? Or do we want to show an error gracefully to the user (because in case of non-unique username, I think we want the latter)

Reply:
You can just manually add a call to Sentry https://docs.sentry.io/platforms/python/usage/